oak_abi/src/lib.rs

```diff
@@ -44,6 +44,16 @@ pub const OAK_LABEL_HTTP_JSON_KEY: &str = "oak-label";
 /// The header key used for protobuf encoded Labels in HTTP requests.
 pub const OAK_LABEL_HTTP_PROTOBUF_KEY: &str = "oak-label-bin";
 
+/// The header key used for JSON formatted signed authentication challenge.
+pub const OAK_SIGNED_CHALLENGE_JSON_KEY: &str = "oak-signed-auth-challenge";
+
+/// The header key used for protobuf formatted signed authentication challenge.
+pub const OAK_SIGNED_CHALLENGE_PROTOBUF_KEY: &str = "oak-signed-auth-challenge-bin";
+
+// TODO(#1357): Remove, or move to tests, when we have a per-connection challenge phrase.
+/// Hash of "oak-challenge", the fixed challenge phrase.
+pub const OAK_CHALLENGE_PHRASE_BASE64_HASH: &str = "o0UQJfxNADV1BLonFxTfWWg7VRVfmoyGpKZ/g67FE2c=";
```

Comment:
I think it would be more readable to actually have the challenge, and compute the hash and base64 at runtime, rather than having an opaque string and explaining how it was generated in a comment. Also the same logic will be necessary in the future for the interactive challenge-response.